### ISSUE TITLE
Review annotations to GO:0001076 'transcription factor activity, RNA polymerase II transcription factor binding'

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#1927]
Dear all, 

For the transcription factor overhaul, we are proposing to obsolete GO:0001076 'transcription factor activity, RNA polymerase II transcription factor binding'. There are not very many annotations, and most proteins are annotated to other MF terms. Please review your annotations: 

https://docs.google.com/spreadsheets/d/1NnkTeU8Ph7p5UHICsDUu94EoSsXbAhDg5lLsDsAs4Po/edit#gid=0

Impacted groups: 

BHF-UCL @RLovering 
ParkinsonsUK-UCL @RLovering 
FlyBase @hattrill 
IntAct @bmeldal 
MGI @ukemi 
SGD @suzialeksander 
TAIR @tberardini 
UniProt @ggeorghiou 
WB @vanaukenk 
ZFIN @sabrinatoro 

Please fill the Google doc and remove your name from the assignees when done. 
-----

**Comment by sabrinatoro** [comment]
ZFIN annotations have been reviewed.

**Comment by RLovering** [comment]
Hi Pascale
I have tried to do this but I am not happy with the definition for one of the recommended terms : GO:0001104 RNA polymerase II transcription cofactor activity
Definition: Interacting selectively and non-covalently with an RNA polymerase II (RNAP II) regulatory transcription factor and also with the RNAP II basal transcription machinery in order to modulate transcription. Cofactors generally do not bind DNA, but rather mediate protein-protein interactions between regulatory transcription factors and the basal RNAP II transcription machinery.

Surprisingly I am not willing to make the assumption that a co-factor TF that binds a specific TF will also bind a basal TF.

There does not seem to be a GO term with the definition (with the text "and also with the RNAP II basal transcription machinery" and "and the basal RNAP II transcription machinery" removed and with the phrase 'specific and other regulatory or basal' added):
Interacting selectively and non-covalently with an RNA polymerase II (RNAP II) regulatory transcription factor  in order to modulate transcription. Cofactors generally do not bind DNA, but rather mediate protein-protein interactions between specific and other regulatory or basal transcription factors.

Basically the expts show a regulatory TF (ie cofactor) binding a specific TF, but no evidence for the cofactor binding to basal TFs. As there could be additional proteins in this complex I am not willing to use the term:GO:0001104 RNA polymerase II transcription cofactor activity. 

The alternative is to just associate the term GO:0140110  transcription regulator activity.

Please advise and I will then revise our annotations

Best

Ruth

**Comment by pgaudet** [comment]
Hi @RLovering 
You're right - GO:0001104 RNA polymerase II transcription cofactor activity
is not appropriate in this case. I agree GO:0140110 transcription regulator activity is better. 

Thanks, Pascale

**Comment by RLovering** [comment]
FYI GO:0140110 transcription regulator activity has a do not annotate restriction

**Comment by RLovering** [comment]
BHF annotations all updated

**Comment by ukemi** [comment]
Assigning to @krchristie  for MGI.

**Comment by vanaukenk** [comment]
WB annotations reviewed

**Comment by tberardini** [comment]
TAIR annotation reviewed.

**Comment by pgarmiri** [comment]
The one UniProt annotation has be reviewed and updated.

**Comment by hdrabkin** [comment]
mgi fixed

**Comment by RLovering** [comment]
sorry ours were done in July but form not completed completely

**Comment by pgaudet** [comment]
@bmeldal can you please check the IntAct annotation ?

**Comment by bmeldal** [comment]
As for Ruth, change was made but I forgot to update the spreadsheet. Sorry.

**Comment by pgaudet** [comment]
No problem. Thanks for checking. 


Pascale
